### DISCUSSION TITLE
Something went wrong. Maybe it is about the path.

### DISCUSSION THREAD

**Discussion by ghost** [stm32duino/Arduino_Core_STM32#2098] [Q&A]
Something went wrong. Maybe it is about the path.

exec: "{runtime.tools.STM32Tools.path}/win/busybox.exe": file does not exist

Compilation error: exec: "{runtime.tools.STM32Tools.path}/win/busybox.exe": file does not exist

<img width="1058" alt="issue20230809" src="https://github.com/stm32duino/Arduino_Core_STM32/assets/123806910/f3c05d6f-6519-4e48-b418-a176fb82084b">

**Comment by fpistm** [comment]
Hi @AHSongyuxuan 
It seems your installation is not correct.
You should try to uninstall and reinstall then check you have the file in the installed directory.
